app/components/pages/Account.jsx

```diff
@@ -9,10 +9,11 @@ class SelectionList extends React.Component {
     const { onChange, checked } = this.props;
     const numericalValue = parseInt(e.target.value, 10);
```

Comment:
Could we rename this to something more meaningful? `toggledId` or something like that?